README.md

```diff
@@ -14,6 +14,8 @@ npm install jest-preset-stylelint jest stylelint --save-dev
 
 ## Setup
 
+### Simple setup
```

Comment:
I think all authors should use the preset as it currently sets one additional property, `testEnvironment: 'node'`, and may set more in the future.

The advanced setup would then be using the preset but overriding the `setupFiles` property.

---

## Setup

Add the preset to your `jest.config.js` or `jest` field in `package.json`:

```json
{
  "preset": "jest-preset-stylelint"
}
```

Optionally, you can avoid specifying `plugins` in every schema by defining your own setup file to configure the `testRule` function. This is useful if you have many tests. There are two steps:

1. Create `jest.setup.js` in the root of your project. Provide [`plugins`](#plugins-arraystring) option to `getTestRule()`:

   ```js
   const stylelint = require("stylelint");
   const getTestRule = require("jest-preset-stylelint/getTestRule");

   global.testRule = getTestRule(stylelint, { plugins: ["./"] });
   ```

2. Add `jest.setup.js` to your `jest.config.js` or `jest` field in `package.json`:

   ```json
   {
     "preset": "jest-preset-stylelint",
     "setupFiles": ["jest.setup.js"]
   }
   ```

---